libs/usb-drive/src/usb_drive.ts

```diff
@@ -100,6 +101,53 @@ async function unmountUsbDrive(mountPoint: string): Promise<void> {
   await exec('sudo', ['-n', join(MOUNT_SCRIPT_PATH, 'unmount.sh'), mountPoint]);
 }
 
+async function formatUsbDrive(
+  devicePath: string,
+  label: string
+): Promise<void> {
+  debug(`Formatting disk ${devicePath} as FAT32 volume with label ${label}`);
+  await exec('sudo', [
+    '-n',
+    join(MOUNT_SCRIPT_PATH, 'format_fat32.sh'),
```

Comment:
I realize this isn't a change that this PR introduced, but `MOUNT_SCRIPT_PATH` being in `../src` assumes that this library is running in place and not packaged up using e.g. `pnpm pack`. Maybe that's fine for now, but I do think it's at least worth adding `format_fat32.sh` to the `files` array in `package.json` and possibly moving it to the package root or a different subdirectory.

Reply:
I think putting it under `scripts` is fine for now. `npm pack` is what would be used if we were, say, publishing this library to the npm registry using `npm publish`.